lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -419,12 +455,76 @@ _computeLoadAndUnloadItems(const UsdPrim& prim)
 
     return itemLabelPairs;
 }
+#if PXR_VERSION >= 2008
+//! \brief Get groups of concrete schema prim types to list dynamically in the UI
+static const std::vector<MAYAUSD_NS_DEF::ufe::SchemaTypeGroup> getConcretePrimTypes(bool sorted)
```

Comment:
The "MAYAUSD_NS_DEF" is only used when declaring a namespace. Here you should just use "MayaUsd::"